setup.py

```diff
@@ -35,6 +35,7 @@
 MODIN_REQUIREMENTS = [
     *PANDAS_REQUIREMENTS,
     f"modin{MODIN_DEPENDENCY_VERSION}",
+    "pandas==2.2.1",
```

Comment:
If Modin installs pandas as well, I don't believe we need the 2.2.1 pin here?

Reply:
Modin 0.28.1 was a release we specially requested, which pins the patch version 2.2.1; modin 0.28.3 only pins minor version 2.2.x. I don't think 2.2.3 (the latest available version) has any breaking changes, but it's not available in snowflake anaconda and might cause problems if we don't pin here.